packages/orm/src/rules.ts

```diff
@@ -87,3 +87,101 @@ function errorMessage(errors: ValidationError[]): string {
     return `Validation errors (${errors.length}): ${message}`;
   }
 }
+
+/**
+ * Creates a validation rule that a field value cannot be smaller than a
+ * certain value.
+ *
+ * @param field The field to validate
+ * @param minValue The inclusive minimum value. e.g 0 range is [0, Infinity]
+ *
+ * @example
+ * // Age cannot be smaller than 0
+ * config.addRule(minValueRule("age", 0));
+ */
+export function minValueRule<T extends Entity, K extends keyof T & string>(
+  field: K,
+  minValue: number,
+): ValidationRule<T> {
+  return (entity) => {
+    const value = entity[field];
+
+    // Ignore undefined and null values
+    if (value !== undefined && value !== null) {
+      // Show an error when the value type is not a number
+      if (typeof value !== "number") {
+        return `${field} must be a number`;
+      }
+
+      // Show an error when the value is smaller than the minimum value
+      if (value < minValue) {
+        return `${field} must be greater than or equal to ${minValue}`;
+      }
+    }
+  };
+}
```

Comment:
I kept getting failing tests when a passing rule would return an `undefined` value. Should a passing rule not return anything?

Reply:
I found out the issue was that I was not running `yarn build` after each change of this file 🙈